front-end/src/screens/OnChain/Treasury/query.ts

```diff
@@ -0,0 +1,63 @@
+import gql from 'graphql-tag';
+
+export const QUERY_LATEST_PROPOSALS = gql`
+  query LatestDemocracyTreasuryProposalPosts($postType: Int!, $postTopic: Int!, $limit: Int! = 5 ) {
+    posts(limit: $limit, where: {
+        type: {
+            id: {
+                _eq: $postType
+            }
+        },
+        topic: {
+            id: {
+                _eq: $postTopic
+            }
+        },
+        onchain_link: {
+            onchain_treasury_proposal_id: {
+                _is_null: false
+            },
+            onchain_referendum_id: {
```

Comment:
treasury proposal never become a referendum, they become a motion, so we need to care about motions here (that's a guess, untested but 99% sure):
```suggestion
            onchain_motion_id: {
```

Reply:
done